### ISSUE TITLE
Mozart: Rename mojo::gfx::composition namespace to mojo::gfx

### ISSUE THREAD

**Issue by j9brown** [domokit/mojo#771]
TL;DR: The extra "composition" part of the namespace isn't adding enough value to be worth keeping.

Looking at mojo::ui for comparison, we find that it isn't too confusing having mojo::ui::InputConnection and mojo::ui::View living in the same namespace today even though these interfaces are managed by different subsystems.  If we had mojo::ui::input::InputConnection and mojo::ui::views::View instead, it wouldn't really add much since clients are expected to use them together most of the time anyhow.

In the mojo::gfx namespace we'll eventually have the compositor, image pipe, display manager, and other services offering core abstractions which complement each other more than they conflict, so scattering them into sub-namespaces doesn't seem all that helpful.

**Comment by j9brown** [comment]
Initially I added the word "composition" because I was concerned that I was using big generic names like Scene, Node, and Resource and I wanted to make it clear that those concepts come from the compositor.

For Scene, it seems obvious but for Node and Resource it might be problematic so we could perhaps rename those.  Mostly worried about what will happen when the namespace has 100 types in it.

Open to debate.  Might decide not to rename this yet.

**Comment by abarth** [comment]
Yeah, the existing names are super verbose on namespaces.

**Comment by j9brown** [comment]
What?  No love for "android.net.wifi.p2p.nsd.WifiP2pDnsSdServiceRequest"  ;)
